.changeset/tidy-orchards.md

```
@@ -0,0 +1,8 @@
+---
+'@ciderpress/cli': patch
+'@ciderpress/config': patch
+'@ciderpress/theme': patch
+'@ciderpress/ui': patch
+---
+
+Remove stale dependency declarations discovered by Knip.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)